packages/arb-token-bridge-ui/src/components/common/SidePanel.tsx

```diff
@@ -7,13 +7,15 @@ type SidePanelProps = {
   isOpen: boolean
   onClose?: () => void
   children: React.ReactNode
+  overridePanelClasses?: string
 }
 
 export const SidePanel = ({
   isOpen,
   heading,
   onClose,
-  children
+  children,
+  overridePanelClasses
```

Comment:
nit: `panelClassName` or `panelClassNameOverrides`

we should also make use of `tailwind-merge`